sklearn/preprocessing/data.py

```diff
@@ -1946,3 +1957,149 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(x, lambda_x=None):
+    if lambda_x is None:
+        x, lambda_x = stats.boxcox(x, lambda_x)
+        return x, lambda_x
+    else:
+        x = stats.boxcox(x, lambda_x)
+        return x
+
+
+def boxcox(X, copy=True):
```

Comment:
Yeah it would be so. But in case you just want the data to be transformed using boxcox transform, this would be helpful right ? Instead of calling ``fit`` and ``transform`` for the same data.